README.md

```diff
@@ -1,68 +1,111 @@
-  clear                                         Clears the stored history of generated request tokens which are used for response validation.
-  generate <type> <reqresp> <attrs> [recieved]  Generate a request or response JWT of any type with attributes in json form. For a response, the optional recieved param is the request
-  validate <response>                           Validate a JWT given in response to an interaction request
-  ```
-
-## Notes
-- The generated requests are JWT encoded and signed.
-- Likewise, the validate command also expects an encoded and signed JWT as input.
-- The seed and password options are not required, however without them the tool will default to a single identity for every user.
-- In future, the tool will detect the presence of a secure hardware element to gather entropy from, making the default identity different for each machine.
-- Support for (signed) credential creation is planned for a future release
+## Intro:  
+
+
+Jolocom-cli is a simple tool for interacting with the Jolocom identity infrastructure from the command line. Please note that currently only a subset of the APIs exposed by the Jolocom Library are supporter. 
+
+The following sections will show how to use the provided tool to anchor a new identity and issue authentication / payment requests. We will also look at a quick example of broadcasting the generated requests to the SmartWallet using deep linking.
+
+## Setting up the CLI:
+
+First, install the CLI tool, either locally or globally:
+
+    npm install -g jolocom-cli
+
+At this point you can already start using it as described in the next sections. For all request signatures a default identity will be used (not recommended for production obviously, but will work for testing). In case you want to create a custom identity you need to provide a custom 32 byte hex encoded seed using the **-i** flag, for example:
+
+    jolocom-cli <command> -i "ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff"
+
+If you actually try to run a command, you'll most likely get the following error:  
+
+
+> current identity is not anchored
+
+This is because the identity was not yet registered on the ledger. To register it, first fuel the key with some Ether to pay for the transaction:
+
+    jolocom-cli fuel -i "ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff"
+
+And after the key has been fueled, anchor it:
+
+    jolocom-cli create -i "ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff"
+
+At this point you can test if it was anchored correctly using:
+
+    jolocom-cli did -i "ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff"
+
+    did: did:jolo:6b8e4245e9863a976b475b7a7c1dc70e290c9fbf0de9eec751d821944658b564
+    created: 2019-04-22T10:09:28.863Z
+
+---
+
+
+# Creating interaction requests:
+
+## Authentication:
+
+The route for authentication is:
```

Comment:
perhaps "The deep link route for authentication on the Smartwallet App is:" for specificity.